gitlab/files/gitlab.rb

```diff
@@ -475,7 +507,13 @@
 gitlab_workhorse['listen_network'] = "tcp"
 # gitlab_workhorse['listen_network'] = "unix"
 # gitlab_workhorse['listen_umask'] = 000
-gitlab_workhorse['listen_addr'] = "127.0.0.1:8181"
+
+{% if server.workhorse.bind is defined -%}
```

Comment:
This will break deployment if server.workhorse is not defined.
You should define defaults in `map.jinja` instead which will save some lines of code.

Reply:
Should've thought about that. I'll move defaults to map.jinja.

Reply:
Fixed (hopefully)!